sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/DoNotPassSameValueAsMultipleArguments.cs

```diff
@@ -44,15 +45,15 @@ public sealed class DoNotPassSameValueAsMultipleArguments :
 
         protected override SyntaxKind InvocationExpressionSyntaxKind =>
             SyntaxKind.InvocationExpression;
-
-        protected override SeparatedSyntaxList<ArgumentSyntax> GetArguments(InvocationExpressionSyntax invocation) =>
+
+        protected override IReadOnlyList<ArgumentSyntax> GetArguments(InvocationExpressionSyntax invocation) =>

```

Comment:
Except this `IReadOnlyList`, has anything else changed?

Reply:
I put a convenience link in the PR comment ;)